hack/quick_install.sh

```diff
@@ -22,6 +22,7 @@ export KNATIVE_OPERATOR_VERSION=v1.15.7
 export KNATIVE_SERVING_VERSION=1.15.2
 export KSERVE_VERSION=v0.14.0
 export CERT_MANAGER_VERSION=v1.16.1
+export GATEWAY_API_VERSION=v1.2.1
```

Comment:
Is there any particular feature that we depend on from this version or can this be lowered?

Reply:
The Timeout field is GA from this version. We are using this to set a default timeout. The timeout value is implementation specific and each of them have their own default values. For example the envoy gateway has a default timeout of 15 secs (If I remember correctly) which is not enough for some of our own tests.